intro/onboarding.md

```diff
@@ -4,15 +4,74 @@ parent: Introduction
+
+# Create a Character
+
+## Character Concepts
+
+Before you begin writing up a character to submit them, it's STRONGLY encouraged to join the #characterbuilding channel, to brainstorm character concepts, refine your ideas, and discuss opportunities for incorporating details into your character's backstory that can tie them into the setting and plot.
+
+We want full three-dimensional characters, with hopes, dreams, flaws, and potential for growth; and who can integrate with ongoing storylines and more generally with the established canon of Destiny Weaves.
+
+We don't expect you to manifest such a character as a fully-formed immaculate virgin birth directly from your forehead. We also don't expect you to know every single detail about Destiny Weaves or Pern in order to ensure your character's personal timeline matches up with our canon.
+What we expect is that you'll be willing to discuss your goals and vision for your character, and to reshape the character concept into a version that suits this community's collaborative story.
+
+## Forum Sub-Account
+
+Each main character you write will be written using a separate sub-account per character, to more easily keep track of which characters are in each scene.
+
+To create a new sub-account, visit he [Sub-Accounts Maangement Page](https://destinyweaves.jcink.net/index.php?act=UserCP&CODE=54) using the link, or by navigating to your user settings, and selecting the section labeled "edit sub-accounts".
+
+Press the "Register a New Account" button on this page, fill out the form, and submit it. The username should be the name of the character you are creating. Submit the form to reach the new-account welcome page (which should state "The administrator wishes to preview all new registered accounts MANUALLY".)
+
+Navigate to the new account's [Profile Information Page](https://destinyweaves.jcink.net/index.php?act=UserCP&CODE=01). Fill out the provided form fields with information for the character, and hit "Amend my profile" to save it.
+
+Navigate to the new account's [Avatar Settings Page](https://destinyweaves.jcink.net/index.php?act=UserCP&CODE=24). Upload or paste a link to an avatar image for your new character, and hit "Update Avatar" to save it.
+
+Navigate to the new account's **public** profile page, and verify that the contents of the profile are what you expected.
+
+Copy the URL of this page (which should be of the form <code>https://destinyweaves.jcink.net/?showuser=<i>number</i></code>), and navigate to the [#app-reviews](https://discord.com/channels/1050092859317035038/1050634202619580447) channel on the Destiny Weaves discord server. Post, in a single message, your chracter's name, a one-sentence summary of the character, and the link to the profile page.
+
+## Character Review
+
+{: note }
+> This is not simply an accept/reject process! Do not submit a character who is so precious you could never bear to change them!
```

Comment:
We should elaborate on this. We're not saying od not to submit a character that you could never change.